UnityProject/Assets/Scripts/Core/Input System/InteractionV2/DefaultWillInteract.cs

```diff
@@ -9,49 +9,55 @@ public static class DefaultWillInteract
 	/// </summary>
 	/// <param name="interaction">interaction to check</param>
 	/// <param name="side">side of the network this is being checked on</param>
+	/// <param name="aps"></param>
 	/// <typeparam name="T">type of interaction</typeparam>
-	public static bool Default<T>(T interaction, NetworkSide side) where T : Interaction
+	public static bool Default<T>(T interaction, NetworkSide side, PlayerTypes aps = PlayerTypes.Normal) where T : Interaction
```

Comment:
why ``aps``?

Reply:
oh need to rename that again, was playerStatesAllowed, guess it should be playerTypesAllowed (pta?)

Reply:
I guess you can add the full name of the parameter to the dosctring